src/pkg/transform/git.go

```diff
@@ -12,8 +12,8 @@ import (
 	"github.com/defenseunicorns/zarf/src/pkg/utils/helpers"
 )
 
-// For further explanation: https://regex101.com/r/YxpfhC/3
-var gitURLRegex = regexp.MustCompile(`^(?P<proto>[a-z]+:\/\/)(?P<hostPath>.+?)\/(?P<repo>[\w\-\.]+?)(?P<git>\.git)?(?P<atRef>@(?P<force>\+)?(?P<ref>[\/\+\w\-\.]+))?(?P<gitPath>\/(?P<gitPathId>info\/.*|git-upload-pack|git-receive-pack))?$`)
+// For further explanation: https://regex101.com/r/YxpfhC/5
+var gitURLRegex = regexp.MustCompile(`^(?P<proto>[a-z]+:\/\/)(?P<hostPath>.+?)\/(?P<repo>[\w\-\.]+?)?(?P<git>\.git)?(\/)?(?P<atRef>@(?P<force>\+)?(?P<ref>[\/\+\w\-\.]+))?(?P<gitPath>\/(?P<gitPathId>info\/.*|git-upload-pack|git-receive-pack))?$`)
```

Comment:
Do we have this tested somewhere in a unit test?

Reply:
more test-cases added. LMK if this is what you had in mind. Thanks!

Reply:
Did you want a plan regex case or more [here](https://github.com/defenseunicorns/zarf/blob/0d0cac8496e397ef63adcc276956b4fcdd50811f/src/pkg/transform/git_test.go#L74)?